backend/routes/passwordRoute.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const router = express.Router();
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
+
+router.post('/check-password', async(req, res) => {
+    const {password} = req.body;
+    const findPassword = await prisma.user.findUnique({
+        where: {password}
+    })
+
+    const exists = findPassword !== null
```

Comment:
make this into a conditional check instead

Reply:
done